### PR TITLE
Update French localization

### PR DESCRIPTION
**Pull request by surfdude29**
As a follow-up to #4611 and #4637 this PR adds translations for new strings, especially some high-visibility starter pack-related ones. It also includes one correction where I had missed changing `Packs` to `Kits`.

I'd be grateful if you could take a look @Signez 🙏

### CODE REVIEW

File: src/locale/locales/fr/messages.po [hunk 1]
```diff
@@ -390,7 +382,7 @@ msgstr "Contenu pour adultes"
 
 #: src/screens/Moderation/index.tsx:356
 msgid "Adult content can only be enabled via the Web at <0>bsky.app</0>."
-msgstr ""
+msgstr "Le contenu pour adultes ne peut être activé que via le Web à <0>bsky.app</0>."
```
Comment (Signez) [inline src/locale/locales/fr/messages.po:385]: We use either `à l'adresse` or `sur` (like English's `on`) for websites references like URLs. The first being longer and less fluid imho, let's use the latter.

```suggestion
msgstr "Le contenu pour adultes ne peut être activé que via le Web sur <0>bsky.app</0>."
```

**Review by Signez — COMMENTED** [review]
Great job, as always! 🚀

File: src/locale/locales/fr/messages.po (hunk 1 above)
Comment (surfdude29) [reply src/locale/locales/fr/messages.po:385]: Ah got it 👍

**Comment by surfdude29** [comment]
Thanks again @Signez, suggestion applied 🙏

**Comment by surfdude29** [comment]
I've added translations for the new strings that have been added recently, including the onboarding tour, the progress guide etc.

I'd be grateful if you could look them over please @Signez 🙏

File: src/locale/locales/fr/messages.po [hunk 2]
```diff
@@ -732,6 +728,10 @@ msgstr "Bluesky"
 msgid "Bluesky is an open network where you can choose your hosting provider. Custom hosting is now available in beta for developers."
 msgstr "Bluesky est un réseau ouvert où vous pouvez choisir votre hébergeur. L’auto-hébergement est désormais disponible en version bêta pour les développeurs."
 
+#: src/components/ProgressGuide/List.tsx:55
+msgid "Bluesky is better with friends!"
+msgstr "Bluesky est meilleur avec des ami·e·s !"
```
Comment (Signez) [inline src/locale/locales/fr/messages.po:733]: I think `entre amis` is more natural than `avec des amis`.

```suggestion
msgstr "Bluesky est meilleur entre ami·e·s !"
```

File: src/locale/locales/fr/messages.po [hunk 3]
```diff
@@ -2505,6 +2567,10 @@ msgstr "Voir le profil du compte"
 msgid "Graphic Media"
 msgstr "Médias crus"
 
+#: src/state/shell/progress-guide.tsx:167
+msgid "Half way there!"
+msgstr "A mi-chemin !"
```
Comment (Signez) [inline src/locale/locales/fr/messages.po:2572]: It's _technically_ a good translation, but I think that translating it more like `nearly there` is more natural in that context.

```suggestion
msgstr "On y est presque !"
```

File: src/locale/locales/fr/messages.po [hunk 4]
```diff
@@ -4294,6 +4385,10 @@ msgstr "Code QR a été téléchargé !"
 msgid "QR code saved to your camera roll!"
 msgstr "Code QR enregistré dans votre photothèque !"
 
+#: src/tours/Tooltip.tsx:111
+msgid "Quick tip"
+msgstr "Conseil rapide"
```
Comment (Signez) [inline src/locale/locales/fr/messages.po:4390]: `Quick tip` is quite a thing by itself; it's not really a `tip` that is `quick`. I prefer using `Petite astuce` that convey the same meaning while being more natural in French.

```suggestion
msgstr "Petite astuce"
```

File: src/locale/locales/fr/messages.po [hunk 5]
```diff
@@ -401,6 +397,10 @@ msgstr "Le contenu pour adultes est désactivé."
 msgid "Advanced"
 msgstr "Avancé"
 
+#: src/state/shell/progress-guide.tsx:177
+msgid "Algorithm training complete!"
+msgstr "Formation à l’algorithme terminée !"
```
[the remark this answers is not in the document]
Comment (surfdude29) [reply src/locale/locales/fr/messages.po]: Yes, I think that's more likely what it means! Thanks for catching that :)

**Comment by surfdude29** [comment]
All suggestions applied, once again many thanks @Signez for your quick and helpful feedback! :)

This is now up-to-date and ready to go @pfrazee 🫡

**Review by pfrazee — APPROVED** [review]
Good deal! Thanks all!

**Pull request merged**